gravitee-apim-gateway/gravitee-apim-gateway-handlers/gravitee-apim-gateway-handlers-shared-policy-group/src/main/java/io/gravitee/gateway/handlers/sharedpolicygroup/policy/SharedPolicyGroupPolicyFactory.java

```diff
@@ -40,13 +40,16 @@ public boolean accept(PolicyManifest policyManifest) {
     }
 
     @Override
-    protected Policy createPolicy(
+    protected HttpPolicy createPolicy(
         ExecutionPhase phase,
         PolicyManifest policyManifest,
         PolicyConfiguration policyConfiguration,
         PolicyMetadata policyMetadata
     ) {
-        Policy policy = new SharedPolicyGroupPolicy(policyMetadata.getName(), (SharedPolicyGroupPolicyConfiguration) policyConfiguration);
+        HttpPolicy policy = new SharedPolicyGroupPolicy(
```

Comment:
should we have this kind of renaming ?
```suggestion
        HttpPolicy policy = new SharedHttpPolicyGroupPolicy(
```

Reply:
`HttpSharedPolicyGroupPolicy` just to keep the 'Http' wording in the same place?

Reply:
Or maybe we can wait to do that when we will migrate this policy to implement HttpPolicy (it still implements the old Policy interface at the moment)?

Reply:
makes sense !